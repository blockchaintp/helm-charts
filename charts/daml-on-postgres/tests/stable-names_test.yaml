suite: daml-on-postgres.test for stable names
templates:
  - daml-service.yaml
  - daml-statefulset.yaml
  - daml-serviceaccount.yaml
release:
  name: test
  namespace: testns
tests:
  - it: check for stable entity names (services, statefulsets, serviceaccounts)
    asserts:
      - matchSnapshot:
          path: metadata.name
  - it: check for stable PVC names
    template: daml-statefulset.yaml
    set:
      postgres:
        persistence:
          enabled: true
    asserts:
      - matchSnapshot:
          path: spec.volumeClaimTemplates
