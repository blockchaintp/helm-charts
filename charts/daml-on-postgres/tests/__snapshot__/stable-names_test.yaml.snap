check for stable PVC names:
  1: |
    - metadata:
        name: pgdata
      spec:
        accessModes:
        - ReadWriteOnce
        resources:
          requests:
            storage: 10Gi
        storageClassName: null
check for stable entity names (services, statefulsets, serviceaccounts):
  1: |
    test-daml-on-postgres
  2: |
    test-daml-on-postgres
  3: |
    test-daml-on-postgres
